applications/asset_tracker_v2/src/modules/gps_module.c

```diff
@@ -181,23 +183,36 @@ static void gps_event_handler(const struct device *dev, struct gps_event *evt)
 	case GPS_EVT_SEARCH_TIMEOUT:
 		LOG_DBG("GPS_EVT_SEARCH_TIMEOUT");
 		SEND_EVENT(gps, GPS_EVT_TIMEOUT);
-		search_stop();
+
+		/* Wrap sending of GPS_EVT_INACTIVE to avoid macro redefinition errors. */
+		inactive_send();
 		break;
 	case GPS_EVT_PVT:
 		/* Don't spam logs */
 		break;
 	case GPS_EVT_PVT_FIX:
 		LOG_DBG("GPS_EVT_PVT_FIX");
 		time_set(&evt->pvt);
-		data_send(&evt->pvt);
-		search_stop();
+		inactive_send();
+
+#if defined(CONFIG_NRF_CLOUD_PGPS)
```

Comment:
In Asset Tracker v1, I added an additional CONFIG to permit this -- the modem guys don't like this feature, especially if the device being tracked sleeps a long time between fixes and travels fast.  The saved location can be pretty far from the current location, so if it is used to assist the GPS unit, it can lead to longer fix times than otherwise.

I am in favor of letting the customer decide whether to do this based on their product's expected use case.  If it is an animal tracker, no problem.  If it is a package tracker which might end up on high speed transportation, don't do this.  If the tracker hardware has an accelerometer, and that accelerometer is used by the application to determine whether the device has been stationary or not, you could do it conditionally. Although, that makes me realize we need a public method to say "forget previous stored location."  I think it is too late to add this for NCS 1.6.0 though.

Reply:
Ok, thank you for the explanation. Added an additional option so that customers can decide if they want this feature or not.